swagger.json

```diff
@@ -1188,26 +1188,18 @@
                 }
             }
         },
-        "/fleet/vehicles/{vehicle_id}/unassigned_driving_segments": {
+        "/fleet/unassigned_driving_segments": {
             "get": {
                 "tags": [
                     "Fleet", "Default"
                 ],
-                "summary": "/fleet/vehicles/{vehicle_id:[0-9]+}/unassigned_driving_segments",
+                "summary": "/fleet/unassigned_driving_segments",
                 "description": "Get the unassigned driving segments for a specified range.",
```

Comment:
nit: could we mention that this returns the unassigned driving segments for all vehicles in the fleet?